docs/roadmap/planning-process.md

```diff
@@ -0,0 +1,28 @@
+# Planning Process
+
+acs-engine features a lightweight process that emphasizes openness and ensures every community member can be an integral part of planning for the future.
+
+## The Role of Maintainers
+
+[Maintainers][] lead the acs-engine project. Their duties include proposing the Roadmap, reviewing and integrating contributions and maintaining the vision of the project.
```

Comment:
We need to include the current roles covered by the "GitHub Issues" Role, and the "GitHub Code Review" role.